.github/workflows/npm-publish-dev.yml

```diff
@@ -0,0 +1,220 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Publish package to GitHub Packages
+on:
+  push:
+    branches: [ "dev" ]
+  workflow_dispatch:
+
+jobs:
+
+  build-and-publish:
+    if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Setup .npmrc file to publish to GitHub Packages
+      - name: Setup Node.js (.npmrc)
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+          registry-url: 'https://npm.pkg.github.com'
+          # Defaults to the user or organization that owns the workflow file
+          scope: 'tazama-lf'
+
+      - name: Version prerelease
+        run: npm version prerelease
+        env:
+          GH_TOKEN: '${{ secrets.GH_TOKEN }}'
+      
+      - name: Install dependencies
+        run: npm ci
+        env:
+          GH_TOKEN: '${{ secrets.GH_TOKEN }}'
+
+      - name: Publish package
+        run: npm publish
+        env:
+          GH_TOKEN: '${{ secrets.GH_TOKEN }}'
+          NODE_AUTH_TOKEN: '${{ secrets.NODE_AUTH_TOKEN }}'
+          
+  sync-version-across-tazama-repos:
```

Comment:
Can't we use dependabot instead?

Reply:
We're not currently doing that, but the change in my opinion should be in the dependabot config so that it's able to update `frms-coe-lib` and `frms-coe-startup-lib`.